guides/common/modules/proc_adding-rhel-system-roles.adoc

```diff
@@ -33,7 +33,7 @@ endif::[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install-project} rhel-system-roles
+# {project-package-install} rhel-system-roles
```

Comment:
I'm reading the guide and everything suggests this should be on the server (so `project`) is correct, but that means we only need to care about RHEL 8, since we no longer run on RHEL 7.

Could you submit that as a separate PR so we can easily cherry pick the fix to the appropriate branches?